client/lib/api/user_preferences.dart

```diff
@@ -101,10 +104,14 @@ class UserPreferences {
 
   Future<bool> setAnalyticsEnabled(bool value) async {
     var analytics = FirebaseAnalytics();
+
     if (!value) {
       await analytics.resetAnalyticsData();
     }
     await analytics.setAnalyticsCollectionEnabled(value);
+    // TODO: ADD BACK PERFORMANCE MONITERING WHEN LEGAL APPROVES
```

Comment:
`TODO(@theswerd)`

that way someone will know to check with you first rather than unilaterally add the performance checks.